src/test/java/com/aspose/imaging/cloud/test/base/ApiTester.java

```diff
@@ -53,99 +55,136 @@
 
     /**
-     * The base URL
+     * The default cloud storage
      */
-    private final String BaseUrl = "http://api.aspose.cloud/";
+    protected final static String DefaultStorage = "Imaging-QA";
     
     /**
-     * The local test folder
+     * Input test files info
      */
-    protected final String LocalTestFolder = "testData";
+    protected static List<StorageFileInfo> InputTestFiles;
 
     /**
-     * The cloud storage test folder
+     * Aspose.Imaging API
      */
-    protected final String CloudTestFolder = "CloudTestJava";
+    protected static com.aspose.imaging.cloud.sdk.api.ImagingApi ImagingApi;
 
     /**
-     * The cloud storage references folder
+     * Aspose.Storage API
      */
-    protected final String CloudReferencesFolder = "CloudTestReferences";
+    protected static com.aspose.storage.api.StorageApi StorageApi;
+	
+	/**
+	 * The server access file
+	 */
+    private final static String ServerAccessFile = "serverAccess.json";
 
     /**
-     * The default cloud storage
+     * The application key
      */
-    protected final String DefaultStorage = "Imaging-QA";
+    private final static String AppKey = "xxx";
 
     /**
-     * The size difference division
+     * The application SID
      */
-    protected final long SizeDiffDivision = 20;
+    private final static String AppSid = "xxx";
 
     /**
-     * Input test files info
+     * The base URL
+     */
+    private final static String BaseUrl = "http://api.aspose.cloud/";
+    
+    /**
+     * Original test data folder
      */
-    protected List<StorageFileInfo> InputTestFiles;
+    private final static String OriginalDataFolder = "CloudTest";
+    
+    /**
+     * Temporary file name
+     */
+    private final static String tempFileName = "imagingJavaSdkTempFile";
 
     /**
-     * Aspose.Imaging API
+     * Gets or sets a value indicating whether resulting images should be removed from cloud storage.
      */
-    protected com.aspose.imaging.cloud.sdk.api.ImagingApi ImagingApi;
+    public Boolean RemoveResult = true;
 
     /**
-     * Aspose.Storage API
+     * Gets or sets a value indicating whether to automatic recover reference (i.e. if resulting images of failed tests are considered as new valid references).
+     * Please, change this value ONLY if you clearly understand the consequences, or it may lead to replacement of the images you need, so they will be lost.
+     */
+    public Boolean AutoRecoverReference = false;
+    
+    /**
+     * The size difference division
      */
-    protected com.aspose.storage.api.StorageApi StorageApi;
+    protected final long SizeDiffDivision = 20;
 
     /**
      * The basic export formats
      */
     protected final String[] BasicExportFormats = new String[]
     {
         "bmp",
-        "jpg",
-        "psd",
-        "tiff",
         "gif",
+        "jpg",
         "png",
-        "j2k",
-        "webp"
+        "psd",
```

Comment:
@MaxCrank Are you sure that need to remove j2k, webp formats from tests?

Reply:
@roxboris please check corporate mail regarding tests across all levels.